Client/Coordinators/Browser/BrowserCoordinator.swift

```diff
@@ -275,17 +275,21 @@ class BrowserCoordinator: BaseCoordinator, LaunchCoordinatorDelegate, BrowserDel
     }
 
     private func showLibrary(with homepanelSection: Route.HomepanelSection) {
-        let navigationController = DismissableNavigationViewController()
-        navigationController.modalPresentationStyle = .formSheet
-        let libraryRouter = DefaultRouter(navigationController: navigationController)
+        if let libraryCoordinator = childCoordinators[LibraryCoordinator.self] {
+            libraryCoordinator.start(with: homepanelSection)
```

Comment:
Yeah, it's not ideal for coordinators. Can we find another way? Maybe saving the state in a user defaults or something else? @OrlaM maybe you have another idea?

Reply:
What kind of state do we need to keep track of? Is it just which segment is selected or is it also scroll location or drilling down into sub-menus etc?

Reply:
If I got this right, the state is a `LibraryPanelMainState`. We keep the state for history and bookmarks mainly (so in what view we're on when the user left the panels).

So this means, sub-menus, if we are in edit mode or folder mode, but not the scroll position. It's a little more intricate than what I first thought.

Reply:
This is a weird pattern. Can we confirm with product if this is actually a desired behaviour or just a by product of how it was originally built.
Since to leave the screen I need to hit done, or drag down from the top to dismiss the screen I would anticipate the next time I go there that it would take me back to the start. How it is now actually feels kind of weird and confusing. If we can remove that requirement it would greatly simplify things and in my opinion be a better UX.

Reply:
I commented on the issue, but we need to keep this. So doing it this way (keeping reference to coordinator to keep the state) is the easiest way for us right now. Further down the line, Orla suggested we could migrate to having Redux for that state instead. That is in a while, so for now this is what we'll have.